rllib/BUILD.bazel

```diff
@@ -1608,6 +1608,85 @@ py_test(
     ],
 )
 
+py_test(
```

Comment:
We follow some hierarhy in this file.

Your tests are in the [Algorithms learning regression tests](https://github.com/HassamSheikh/ray/blob/master/rllib/BUILD.bazel#L158-L165).

Within this group, You can see that [APPO](https://github.com/HassamSheikh/ray/blob/master/rllib/BUILD.bazel#L167) is first collection of tests. Envs for the algorithm follows right under (in the alphabetical order).

Add APPO with Footsies test just above APPO with [MultiAgentCartPole](https://github.com/HassamSheikh/ray/blob/master/rllib/BUILD.bazel#L241).